### PR TITLE
[WaveTransform] Disabling the LWT for amdgpu-num-agpr.ll test

### PR DESCRIPTION
**Pull request by vg0204**
In this test file, stress testing happens with the VGPRs and AGPRs
number which forces the desired warning checks for the legacy flow.

The same cannot be achieved simply for the LWT flow due to the edge case
scenarios working together with the VGPRs reservation for WWM registers
allocation that triggers the insufficincy error. So, this test cases
sets LWT=0 for now & later need to be reconsidered for modification as per
its intent for the LWT flow.

### CODE REVIEW

File: llvm/test/CodeGen/AMDGPU/amdgpu-num-agpr.ll [hunk 1]
```diff
@@ -9,6 +9,11 @@
 ; The occupancy target warnings should be a side effect of violating
 ; the register budget with asm.
 
+; FIXME: This test crashes with the error of not finding enough VGPRs
```
Comment (cdevadas) [inline llvm/test/CodeGen/AMDGPU/amdgpu-num-agpr.ll]: Instead of generic FIXME, add them under ; TODO-WAVETRANSFORM:
In that way they stand out and we can address them later before upstreaming the wave transform project.

**Review by cdevadas — APPROVED** [review]
